src/Configuration/FormatterConfiguration.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Configuration;
+
+class FormatterConfiguration
+{
+    /** @var array<string, array<mixed>> */
+    private $config;
+
+    /**
+     * @param array<string, array<mixed>> $config
+     */
+    public function __construct(array $config)
+    {
+        $this->config = $config;
+    }
+
+    /**
+     * @return array<mixed>
+     */
+    public function getConfigFor(?string $area): array
```

Comment:
I'd rather have something like `getConfig()` and `getConfigFor(string $area)` instead of a nullable argument.

Reply:
Done!